public/video-ui/src/components/VideoItem/index.jsx

```diff
@@ -46,6 +45,7 @@ export default class VideoItem extends React.Component {
 
   render() {
     const video = this.props.video;
+    const mediaPlatform = VideoUtils.getMediaPlatform(video);
```

Comment:
note the 'video' here doesn't have all the video fields, just the fields returned from the mediaAtomSummary endpoint